elastalert/alerts.py

```diff
@@ -361,9 +361,13 @@ def find_existing_ticket(self, matches):
         else:
             title = self.create_title(matches)
 
+        if 'jira_ignore_in_title' in self.rule:
+            title = title.replace(matches[0].get(self.rule['jira_ignore_in_title'], ''), '')
+
         # This is necessary for search for work. Other special characters and dashes
```

Comment:
`This is necessary for search *for* work` or `This is necessary for search *to* work`?